pandas/tseries/tests/test_timeseries.py

```diff
@@ -2551,6 +2551,15 @@ def test_intersection_bug_1708(self):
         result = index_1 & index_2
         self.assertEqual(len(result), 0)
 
+    def test_union_freq_both_none(self):
+        #GH11086
```

Comment:
assert that the ``result.freq`` is ``None`` (in case the original is actually modified)